packages/uswds-core/src/styles/mixins/helpers/at-media.scss

```diff
@@ -19,6 +21,19 @@
     @media all and (min-width: #{$bp}) {
       @content;
     }
+  } @else if map.has-key($custom-breakpoints, $quoted-bp) {
+    @if unit(map.get($custom-breakpoints, $quoted-bp)) == "px" {
+      @if $theme-respect-user-font-size {
+        $bp: px-to-user-em(map.get($custom-breakpoints, $quoted-bp));
+      } @else {
+        $bp: map.get($custom-breakpoints, $quoted-bp);
+      }
+      @media all and (min-width: #{$bp}) {
+        @content;
+      }
+    } @else {
+      @warn 'Custom breakpoint `#{$quoted-bp}` has the unit #{unit(map.get($custom-breakpoints, $quoted-bp))} and can not be generated. Custom breakpoints must be set in px.';
```

Comment:
**suggestion (non-blocking)**: Thinking it would be good to reference the related setting in the warning message. Small tweak, but I think it will make troubleshooting just a bit easier. 

Current error message:
![image](https://github.com/user-attachments/assets/520a3f17-5339-4d64-acae-ed9a5e631a42)

```suggestion
      @warn 'Custom breakpoint `#{$quoted-bp}` was defined with the unit #{unit(map.get($custom-breakpoints, $quoted-bp))} and can not be generated. Values for `$theme-utility-breakpoints-custom` must be set with `px` units.';
```